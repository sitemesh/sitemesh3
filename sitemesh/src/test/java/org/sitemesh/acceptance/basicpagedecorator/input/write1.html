<html>
	<head>
		<title>Hello 4</title>
		<meta name="test" content="TEST">
	</head>
	<body id="write1" class="write-class-1">
	    <p>Property write test</p>
	</body>
</html>