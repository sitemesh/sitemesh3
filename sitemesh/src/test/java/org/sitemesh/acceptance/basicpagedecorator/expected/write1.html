<!-- Common header -->
<html>
  <head>
    <title>Hello 4</title>
    <meta name="test" content="TEST">
  </head>
  <body id="write1" class="write-class-1 TEST">
	<input id='meta.test' type='hidden' value='TEST' />
    <h1 style="text-align:center">Hello 4</h1>

    <!-- body -->
    <p>Property write test</p>
    <!-- end of body -->

  </body>
</html>
<!-- Common footer -->